libs/vkd3d/resource.c

```diff
@@ -1263,6 +1264,22 @@ uint32_t vkd3d_view_entry_hash(const void *key)
             hash = hash_combine(hash, k->u.texture.allowed_swizzle);
             break;
 
+        case VKD3D_VIEW_TYPE_SAMPLER:
+            hash = (uint32_t)k->u.sampler.Filter;
+            hash = hash_combine(hash, (uint32_t)k->u.sampler.AddressU);
+            hash = hash_combine(hash, (uint32_t)k->u.sampler.AddressV);
+            hash = hash_combine(hash, (uint32_t)k->u.sampler.AddressW);
+            hash = hash_combine(hash, float_bits_to_uint32(k->u.sampler.MipLODBias));
+            hash = hash_combine(hash, (uint32_t)k->u.sampler.MaxAnisotropy);
+            hash = hash_combine(hash, (uint32_t)k->u.sampler.ComparisonFunc);
+            hash = hash_combine(hash, float_bits_to_uint32(k->u.sampler.BorderColor[0]));
```

Comment:
Might be good to ignore BorderColor if app leaves it uninitialized and no address uses border color.

Reply:
Fixed.